### PR TITLE
Update/bifi

### PR DESCRIPTION
**Pull request by tlatkdgus1**
##### List of audit links if any: 
https://github.com/bifrost-platform/BIFI/blob/master/docs/ENG/(ENG)_BiFi_BIFROST_Extension_Theori.pdf
https://github.com/bifrost-platform/BIFI/blob/master/docs/ENG/(ENG)_BiFi_Lending_Sooho_EN_Public.pdf
https://github.com/bifrost-platform/BIFI/blob/master/docs/ENG/(ENG)_BiFi_Token_Hexlant_EN.pdf
https://github.com/bifrost-platform/BIFI/blob/master/docs/ENG/(ENG)_BiFrost_BTC_Extension_CertiK.pdf
https://github.com/bifrost-platform/BiFi-X/blob/main/docs/bifrost_bifix_audit.pdf

##### Github org/user (Optional, if your code is open source, we can track activity):
https://github.com/bifrost-platform/BIFI

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/bifi exports TVL: 
        
 
 
    bitcoin                   22.90 M
    bfc                       6.10 M
    ethereum                  1.05 M
    bsc                       831.75 k
    klaytn                    125.72 k
    avax                      101.12 k
    ethereum-staking          34.17 k
    staking                   34.17 k
    
    total                    31.11 M

**Comment by tlatkdgus1** [comment]
Hi Team! I have question.

our stable coin not registered yet in coingecko

so,  I token mapped btcusd -> usdc

I guess this issue related about my another PR
> https://github.com/DefiLlama/DefiLlama-Adapters/pull/9888#issuecomment-2065743951

File: projects/helper/tokenMapping.js [hunk 1]
```diff
@@ -112,6 +112,9 @@ const fixBalancesTokens = {
   bsquared: {
     '0x0000000000000000000000000000000000000000': { coingeckoId: "bitcoin", decimals: 18, },
   },
+  bfc: {
+    '0x6906Ccda405926FC3f04240187dd4fAd5DF6d555': { coingeckoId: "usd-coin", decimals: 18, },
```
Comment (g1nt0ki) [inline projects/helper/tokenMapping.js:116]: hi @tlatkdgus1 most of the PR is good, except for this part. BtcUSD is not USDC, can wait for coingecko listing before adding it here, another option is, if you share some uni v2/v3 pool with enough liquidity, we can use it to price it in our server.

**Review by g1nt0ki — CHANGES_REQUESTED** [review]
.

**Comment by g1nt0ki** [comment]
> Hi Team! I have question.
> 
> our stable coin not registered yet in coingecko
> 
> so, I token mapped btcusd -> usdc
> 
> I guess this issue related about my another PR
> 
> > [#9888 (comment)](https://github.com/DefiLlama/DefiLlama-Adapters/pull/9888#issuecomment-2065743951)

missed this comment, sorry, dont like the idea of linking to usdc gecko id

**Comment by g1nt0ki** [comment]
merged here: https://github.com/DefiLlama/DefiLlama-Adapters/pull/10246

**Pull request closed without merging**